docs/reference/05-create.md

```diff
@@ -0,0 +1,83 @@
+# Create records
+
+Find, update and delete records.
+
+The examples use the following prisma schema:
+
+```prisma
+model Post {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    published Boolean
+    title     String
+    content   String?
+
+    comments Comment[]
+}
+
+model Comment {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    content   String
+
+    post   Post @relation(fields: [postID], references: [id])
+    postID String
+}
+```
+
+### Create a record
+
+```go
+created, err := client.Post.CreateOne(
+    // required fields
+    Post.Title.Set("what up"),
+    Post.Desc.Set("this is a description"),
+
+    // optional fields
+    Post.ID.Set("id"),
+    Post.Title.Set("name"),
+    Post.Stuff.Set("stuff"),
+).Exec(ctx)
+```
+
+### Create a record with a relation
+
+Use the method `Link` to connect new objects with existing ones. For example, the following query creates a new post and sets the postID attribute of the comment.
```

Comment:
Use the `Link` method. 

We've been using the word records to talk about results, so objects => records.